portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -6,21 +6,68 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.Info;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+    String text = Jsoup.clean(request.getParameter("text-input"), Whitelist.none());
+    long timestamp = System.currentTimeMillis();
+
+    System.out.println("You submitted: " + text);
+
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Task");
+    FullEntity taskEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("title", text)
+            .set("timestamp", timestamp)
```

Comment:
I would use constants such as  TITLE_FIELD, TIMESTAMP_FIELD or something similar to help avoid mistakes. This offers a few advantages:
- If you ever need to change timestamp in the schema to something different, you only need to change it once.
- In the event you do need to change everything. Your program will complain loudly if you don't replace all your variables properply. e.g. you using the field TITLE_FIELD, and you end up changing it to HEADING_FIELD. With what you have now, and using constants, you need to find every instance and replace it with the new variable name or string, but if you use constants and you forget to change TITLE_FIELD - it will loudly complain to you during compile time that TITLE_FIELD doesn't exist (assuming you replaced it with HEADING_FIELD). vs. using strings you likely will encounter a run time error (likely will cause you more headache when debugging)